SCClassLibrary/Common/GUI/Base/QWindow.sc

```diff
@@ -96,14 +96,12 @@ Window {
 	}
 
 	/* NOTE:
-	- 'server' is only for compatibility with SwingOSC
 	- all args have to be of correct type for Widget constructor to match!
 	*/
 	*new { arg name="",
 		bounds,
 		resizable = true,
 		border = true,
-		server,
```

Comment:
Would you mind to revert this one change?

It's risky to delete arguments from method signatures. User code may expect to provide the `scroll` flag as the 6th value -- by deleting one argument, that's moved to the 5th position. It's indeed odd to retain an argument that is never used currently, but for backward compatibility, I'd have to suggest that a proper fix is out of scope of this PR.

Reply:
I can remove this, but in my experience it is better to delete stuff that is not necessary anymore as otherwise there will never be the time for deletion which is unavoidable.

Reply:
I agree that removing the argument is likely to break code.

We have a deprecation procedure which we could try to follow here, even though it isn't straightforward. You could change the name to something like `deprecatedDummyArgument`. And then add a check:
```
if(deprecatedDummyArgument.notNil) { 
    "this argument is deprecated, it was kept for SwingOSC compatibility, but will be removed".warn;
};
```



For that: 
The Cocoa references should be removed in a separate pull request. One could deprecate them by adding a `*new` method to each:

```
*new {
    "this class is deprecated, it was kept for SwingOSC compatibility, but will be removed".warn;
    ^this.deprecated(thisMethod);
}


But perhaps that is too much trouble? I am not sure – what do you think @jamshark70 ?

Reply:
Agreed to deprecate the argument name.

I think it's better if ".11," ".12" etc releases avoid changing method signatures, especially for argument positions. We've been lax about it in the past and it always causes unexpected user pain.

Reply:
I re-introduced the argument with a deprecation warning - hope it works this way.